lib/api/core/models/repositories/userRepository.js

```diff
@@ -27,6 +27,7 @@ const
   uuid = require('node-uuid'),
   User = require('../security/user'),
   Repository = require('./repository'),
+  BadRequestError = require('kuzzle-common-objects').errors.BadRequestError,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use destructuring syntax for these assignments from "require('kuzzle-common-objects').errors". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3514)